docs/lib/datastore/fragments/ios/data-access/query-basic-snippet.md

```diff
@@ -17,11 +17,11 @@ Query has an alternative signature that allows to fetch a single item by its `id
 ```swift
 Amplify.DataStore.query(Post.self, byId: "123") {
     switch $0 {
-        case .success(let result):
-            // result will be a single object of type Post or nil
-            print("Posts: \(result)")
-        case .failure(let error):
-            print("Error listing posts - \(error.localizedDescription)")
+    case .success(let result):
+        // result will be a single object of type Post or nil
+        print("Posts: \(result)")
+    case .failure(let error):
+        print("Error listing posts - \(error.localizedDescription)")
```

Comment:
Instead of using the word "listing", maybe it should be something like "Error on query() for type Post"